lib/matplotlib/axes/_axes.py

```diff
@@ -5801,9 +5801,11 @@ def hist(self, x, bins=None, range=None, normed=False, weights=None,
         normed : boolean, optional
             If `True`, the first element of the return tuple will
             be the counts normalized to form a probability density, i.e.,
-            ``n/(len(x)`dbin)``, i.e., the integral of the histogram will sum
-            to 1. If *stacked* is also *True*, the sum of the histograms is
-            normalized to 1.
+            the integral of the histogram will sum to 1. So, the normalized
```

Comment:
Ok, Thomas, no problem flipping it. Anyway, these two normalization methods of histograms are somehow like miles and kilometers, in the sense that you can say 'Hey, this is not in kilometers, this is in miles'. I can also switch 'the integral of the histogram will sum to 1' to 'the area (or integral) under the histogram will sum to one'.